arduino/resources/helpers.go

```diff
@@ -41,12 +41,12 @@ func (r *DownloadResource) IsCached(downloadDir *paths.Path) (bool, error) {
 	if err != nil {
 		return false, fmt.Errorf("getting archive path: %s", err)
 	}
-
-	if exist, err := archivePath.Exist(); err != nil {
+	exist, err := archivePath.Exist()
+	if err != nil {
 		return false, fmt.Errorf("checking archive existence: %s", err)
-	} else {
-		return exist, nil
 	}
+
+	return exist, nil
 }
```

Comment:
I don't know if I like better this syntax or the previous one.... but if the linter says that this is better... :-)